README.md

```diff
@@ -87,6 +88,13 @@ Refer to the gradle doc for [PmdExtension](https://docs.gradle.org/current/dsl/o
 
 The default `pmdRuleSetFile`, `findBugsExcludeFilterFile`, and `lintConfig` files can be found [here](src/main/resources).
 
+### Publishing
+This project is set to publish to the [Gradle plugins repository](https://plugins.gradle.org/). To publish an update to the plugin, add the following lines to the home directory `gradle.properties` file:
+```
+gradle.publish.key=#######
+gradle.publish.secret=#######
+```
+and then run `./gradlew publishPlugins`
```

Comment:
Let's also mention that if you don't already have the key/secret, you can use `./gradlew login` to log into the gradle site via the web and automatically add them that way.